doc/developer-guides/hld/hld-virtio-devices.rst

```diff
@@ -76,39 +76,39 @@ highlight several key virtio concepts important to ACRN:
 Frontend virtio driver (FE)
   Virtio adopts a frontend-backend architecture that enables a simple but
   flexible framework for both frontend and backend virtio drivers. The FE
-  driver merely needs to offer services configure the interface, pass messages,
-  produce requests, and kick backend virtio driver. As a result, the FE
+  driver merely needs to offer services that configure the interface, pass messages,
+  produce requests, and kick the backend virtio driver. As a result, the FE
   driver is easy to implement and the performance overhead of emulating
   a device is eliminated.
 
 Backend virtio driver (BE)
-  Similar to FE driver, the BE driver, running either in user-land or
+  Similar to the FE driver, the BE driver, running either in userland or
   kernel-land of the host OS, consumes requests from the FE driver and sends them
```

Comment:
If we're changing user-land to userland, we should change kernel-land to kernelland right?  Although "kernelland" looks odd because of the double "l" so maybe best to leave that as is...